src/ethereum/vm/ops.py

```diff
@@ -13,14 +13,103 @@
 implementations.
 """
 
-from .instructions import add, push1, sstore
+from . import instructions
 
+# Arithmetic Operations
+STOP = 0x00
 ADD = 0x01
+MUL = 0x02
+SUB = 0x03
+DIV = 0x04
+SDIV = 0x05
+MOD = 0x06
+SMOD = 0x07
+ADDMOD = 0x08
+MULMOD = 0x09
+EXP = 0x0A
+SIGNEXTEND = 0x0B
```

Comment:
Should we use `Enum` here?

Reply:
cc @SamWilsn

Reply:
One thing to note with `Enum` is that we'll have to catch `ValueError` and treat it as an invalid opcode, but I think that's reasonable.

Reply:
I know that I kind of mentioned about the ENUMs, but would it be ok if I implemented this in #60 . I'd be doing a bit of restructuring the opcode functions into their corresponding files over there.